### DISCUSSION TITLE
Feature idea: snap a photo and have AI put a title, tags etc

### DISCUSSION THREAD

**Discussion by abdessalaam** [karakeep-app/karakeep#391] [Ideas]
Often when reading I would like to save a quote from a book, either printed or online.

Would there be a way to snap a photo of a book page using Hoarder app (or share to it from the camera app) and let it save it, title it, tag it and perhaps OCR with the power of AI? 

Maybe this could also work for screenshots shared from a mobile reader app...

**Comment by MohamedBassem** [comment]
Hey, you can already do exactly that once you enable hoarder in the share sheet :)

![image](https://github.com/user-attachments/assets/5a2a0771-4fae-49fc-8981-7b596d517a56)

**Reply by abdessalaam** [reply]
Assalamu aleykum akhi, thank you! The ai tagging of pictures works like a charm! 
However, I was hoping for a bit more than that: an OCR (maybe triggered by a button to save resources?) that would actually transcribe the entire  text from the picture,  and also add a title... Would that be possible, you think?

**Reply by MohamedBassem** [reply]
@abdessalaam ah yes, this is planned in #296

**Reply by abdessalaam** [reply]
Great, thanks!
